assets/js/components/icons/stories/index.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import {
+	IconAllReviews,
+	IconCheckChecked,
+	IconCheckUnchecked,
+	IconTrash,
+	IconFolder,
+	IconFolderStar,
+	IconProductOnSale,
+	IconProductSearch,
+	IconProductTag,
+	IconNewReleases,
+	IconRadioSelected,
+	IconRadioUnselected,
+	IconReviewsByProduct,
+	IconReviewsByCategory,
+	IconWidgets,
+	IconWoo,
+	IconMoney,
+	IconExternal,
+} from '../';
```

Comment:
You may want to leave this story out of this pull because the work happening in #1644 will invalidate all of this (and create tough to deal with conflicts). I think the examples you've got in this pull are sufficient enough to get the ball rolling.

Reply:
Good point, removed (along with `ProductPreview` story) in ddbf2843593e622e71b4cdde38bee9239ce4743d.